R/ResamplingBootstrap.R

```diff
@@ -70,19 +67,15 @@ ResamplingBootstrap = R6Class("ResamplingBootstrap", inherit = Resampling,
       rownames(M) = NULL
       list(row_ids = ids, M = M)
     },
-
```

Comment:
I note there are a lot of these line breaks removed with `strict = FALSE`. This is due to `styler:::remove_line_break_in_fun_call()`, which removes empty lines in function calls, but only if no comment is around, as discussed in https://github.com/r-lib/styler/issues/632 and https://github.com/r-lib/styler/issues/627. We can either remove this rule or set a minimal number of lines that are required in order to assume this is a *big call* and the line breaks are used to *group* different arguments (similar to empty lines we insert at the top of a function declaration if the body of the function has more than k lines) and keep the blank lines for these situations.

Reply:
Extra newlines normally serve a purpose, I would also opt to disable this.

Reply:
Addressed in https://github.com/mlr-org/styler.mlr/issues/10.

Reply:
🎉